saleor/graphql/checkout/tests/mutations/test_checkout_delivery_method_update.py

```diff
@@ -1461,3 +1462,729 @@ def test_checkout_delivery_method_update_external_shipping_triggers_webhooks(
 
     tax_delivery = tax_delivery_call.args[0]
     assert tax_delivery.webhook_id == tax_webhook.id
+
+
+@mock.patch(
+    "saleor.graphql.checkout.mutations.utils.call_checkout_info_event",
+    wraps=call_checkout_info_event,
+)
+@mock.patch(
+    "saleor.graphql.checkout.mutations.utils.invalidate_checkout",
+    wraps=invalidate_checkout,
+)
+@mock.patch("saleor.webhook.transport.synchronous.transport.send_webhook_request_sync")
+def test_checkout_delivery_method_update_from_cc_to_external_shipping(
+    mocked_send_webhook_request_sync,
+    mocked_invalidate_checkout,
+    mocked_call_checkout_info_event,
+    settings,
+    checkout_with_delivery_method_for_cc,
+    shipping_app,
+    api_client,
+):
+    # given
+    checkout = checkout_with_delivery_method_for_cc
+    settings.PLUGINS = ["saleor.plugins.webhook.plugin.WebhookPlugin"]
+    response_method_id = "abcd"
+    response_shipping_name = "Provider - Economy"
+    response_shipping_price = "10"
+    mock_json_response = [
+        {
+            "id": response_method_id,
+            "name": response_shipping_name,
+            "amount": response_shipping_price,
+            "currency": "USD",
+            "maximum_delivery_days": "7",
+        }
+    ]
+    mocked_send_webhook_request_sync.return_value = mock_json_response
+
+    method_id = graphene.Node.to_global_id(
+        "app", f"{shipping_app.id}:{response_method_id}"
+    )
+
+    # when
+    response = api_client.post_graphql(
+        MUTATION_UPDATE_DELIVERY_METHOD,
+        {"id": to_global_id_or_none(checkout), "deliveryMethodId": method_id},
+    )
+
+    # then
+    data = get_graphql_content(response)["data"]["checkoutDeliveryMethodUpdate"]
+    errors = data["errors"]
+    assert not errors
```

Comment:
Shouldn't we also check the response from the query? 
It's important to have a valid data on DB, but we should ensure that proper data are returned in the response as well.

Reply:
The comment concerns all added tests in this file